bazelisk.go

```diff
@@ -387,22 +407,22 @@ func migrate(bazelPath string, baseArgs []string, newArgs []string) {
 
 	// 2. Try with no flags, as a sanity check.
 	args = baseArgs
-	fmt.Printf("\n\n--- Running Bazel with no incompatible flags\n\n", args)
+	fmt.Printf("\n\n--- Running Bazel with no incompatible flags\n\n")
 	fmt.Printf("bazel %s\n", strings.Join(args, " "))
 	exitCode, err = runBazel(bazelPath, args)
 	if err != nil {
 		log.Fatalf("could not run Bazel: %v", err)
 	}
 	if exitCode != 0 {
 		fmt.Printf("Failure: Command failed, even without incompatible flags.\n")
-		os.Exit(0)
+		os.Exit(1)
```

Comment:
Can we do `os.Exit(exitCode)` here to preserve Bazel's exit code?

Reply:
Good point, done.